distribution/lib/Standard/Table/0.0.0-dev/src/Data/Data_Formatter.enso

```diff
@@ -27,3 +41,54 @@ from Standard.Base import all
    - true_values: Values representing True.
    - false_values: Values representing False.
 type Data_Formatter trim_values:Boolean=True allow_leading_zeros:Boolean=False decimal_point:Text='.' thousand_separator:Text='' datetime_formats:[Text]=["yyyy-MM-dd HH:mm:ss"] date_formats:[Text]=["yyyy-MM-dd"] time_formats:[Text]=["HH:mm:ss"] locale:Locale=Locale.default true_values:[Text]=["True","true","TRUE"] false_values:[Text]=["False","false","FALSE"]
+
+## PRIVATE
+Data_Formatter.get_thousand_separator = if this.thousand_separator.is_empty then Nothing else this.thousand_separator
```

Comment:
The Java side copes fine with `Nothing` so don't think we need this

Reply:
But that's a different thing - in our case `''` is permitted to mean what Java will want as `null`. So yeah, I know that Java copes with Nothing and that's why I convert empty string to Nothing. To make the Java interface a bit simpler (not having to handle the two variants).